RussianFilter/sections/specific.txt

```diff
@@ -60,6 +60,7 @@ sevastopol.su,zn.ua,pravo.ru,lubertsyriamo.ru,kommersant.ru,dvnovosti.ru,kubnews
 !
 !
 !
+mel.fm###zen-group-iframe
```

Comment:
Not an ad and you can block all these elements with a single rule.
Fixed https://github.com/AdguardTeam/AdguardFilters/commit/1d675c35a6261330f7aa6a9fc583722b377620be